templates/database-view-wic.html.j2

```diff
@@ -7,6 +7,18 @@
 <div id="filter-dialog-box" title="Filters">
     <form name="filter-settings" id="filter-settings" method="post">
         <fieldset>
+            <div>
```

Comment:
Since we repeat this, I added a folder called widgets in the template directory. Stuff like should should ideally have it's own file so we can simply {{ include "widgets/widget" }}